main.css

```diff
@@ -0,0 +1,62 @@
+h1 {
+font-family: Arial, Helvetica, sans-serif;
```

Comment:
Just a styling comment, make sure the code inside your brackets is indented one level!